### DISCUSSION TITLE
Can pg_duckdb be installed on Windows?

### DISCUSSION THREAD

**Discussion by enstarbob** [duckdb/pg_duckdb#169] [Q&A]
Both PostgreSQL and DuckDB are installable on Windows, but the installation guidance for pg_duckdb doesn't mention Windows.

Is this just a documentation oversight and 'of course you can', or is there some extra limitation currently preventing it running on Windows?

Either way, it would be really useful if it could! Thank you

**Comment by JelteF** [comment]
It might "just work", but we don't have tests for windows so I would be quite surprised. Feel free to contribute windows support though.
